### PR TITLE
Attempting to debug broken Hugo build

### PR DESCRIPTION
**Pull request by rebeccabilbro**
### Scope of changes

We have a Hugo build issue with the new theme that has to do with [this error](https://github.com/wowchemy/wowchemy-hugo-themes/issues/1665) but that is not solved by using the proposed solution - the problem persists even with the extended version of Hugo installed. I have no idea how to diagnose.

Resolves SC-17358

### Type of change

- [ ] new feature
- [ ] bug fix
- [x] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria

Describe how reviewers can test this change to be sure that it works correctly. Add a checklist if possible.

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [ ]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [ ]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: .github/workflows/docs.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup Hugo
         uses: peaceiris/actions-hugo@v2
         with:
-          hugo-version: '0.101.0'
+          hugo-version: '0.111.0'
```
Comment (rebeccabilbro) [inline .github/workflows/docs.yaml:22]: Note that if there's still an error, we'll only see it after we merge the branch and `hugo --minify -s docs/` runs

File: .github/workflows/docs.yaml [hunk 2]
```diff
@@ -5,30 +5,48 @@ on:
       - main
     tags:
       - 'v*'
+  pull_request:
```
Comment (bbengfort) [inline .github/workflows/docs.yaml:8]: I added this so we could test the action in this PR … but I'd like to remove it before we merge.

File: .github/workflows/docs.yaml [hunk 3]
```diff
@@ -5,30 +5,49 @@ on:
...
+
+      - name: Install NPM Dependencies
+        run: npm install
+
+      - name: Update Hugo Mod
+        run: hugo mod clean && hugo mod get -u ./... && hugo mod tidy
+
       - name: Build
-        run: hugo --minify -s docs/
+        run: hugo --gc --minify --templateMetrics --templateMetricsHints --verbose --verboseLog --buildDrafts --buildExpired --buildFuture --forceSyncStatic || hugo --themesDir ../.. --gc --minify --templateMetrics --templateMetricsHints --verbose --verboseLog --buildDrafts --buildExpired --buildFuture --forceSyncStatic
```
Comment (bbengfort) [inline .github/workflows/docs.yaml:45]: These commands were from the theme `package.json` -- not sure what they do, but adding them to see if it helps 🤷🏽

File: .github/workflows/docs.yaml [hunk 4]
```diff
@@ -5,30 +5,51 @@ on:
...
 
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18.x.x"
+          cache: 'npm'
+          cache-dependency-path: '**/package-lock.json'
+
+      - name: Install NPM Dependencies
+        run: npm install
```
Comment (rebeccabilbro) [inline .github/workflows/docs.yaml:41]: I wonder if we need to specify npm version too? It looks like they just released a new version 4 days ago https://www.npmjs.com/package/npm?activeTab=versions

File: docs/go.sum [hunk 5]
```diff
@@ -1,64 +1,65 @@
...
+github.com/gethugothemes/hugo-modules/shortcodes/video v0.0.0-20230522112131-f98e6d34f6a4/go.mod h1:s5CHMBd7/NalnckFfRhZJvqXPY/QTrluDvRgZm7+Cr0=
+github.com/gethugothemes/hugo-modules/shortcodes/vimeo-lite v0.0.0-20230522112131-f98e6d34f6a4 h1:bz1yB9ZItEdbREhjXz7cGy7lNsRX6/ilj9bKcMRwMIU=
+github.com/gethugothemes/hugo-modules/shortcodes/vimeo-lite v0.0.0-20230522112131-f98e6d34f6a4/go.mod h1:Ac9Ell5o+JV5DZ9dBgW767ZcSvoz/x+2UuFLvMoThwk=
+github.com/gethugothemes/hugo-modules/shortcodes/youtube-lite v0.0.0-20230522112131-f98e6d34f6a4 h1:CfkQo+i2J7jvUujd2OTGYSYE3uwI+q6/roKFncmtNXk=
+github.com/gethugothemes/hugo-modules/shortcodes/youtube-lite v0.0.0-20230522112131-f98e6d34f6a4/go.mod h1:DtWNw6au8NQg1Y0x470DstCHqjWZPnwm61Uu6ZIMy50=
 github.com/gohugoio/hugo-mod-bootstrap-scss/v5 v5.20200.20101 h1:wUe8RXjpJYs83CucuWJT7XuqCzQ52f5/N7zlV8woHEg=
 github.com/gohugoio/hugo-mod-bootstrap-scss/v5 v5.20200.20101/go.mod h1:Z5Bhe4pfn8reoHhWqCQMWEAynLCIeBAt1TDwni4fipM=
+github.com/gohugoio/hugo-mod-bootstrap-scss/v5 v5.20300.20002 h1:6F6sjCSkFIIgqLp4SEIL1Ao9ZfKyM6OmgqEdTlc58Yk=
+github.com/gohugoio/hugo-mod-bootstrap-scss/v5 v5.20300.20002/go.mod h1:Q6TrjM78wB13zkOMBSV4IZIZ54tbshmE2fem9CYx2pU=
```
Comment (rebeccabilbro) [inline docs/go.sum:62]: I was finally able to get this to build locally by downgrading:

```hugo mod get github.com/gohugoio/hugo-mod-bootstrap-scss/v5@v5.20200.20101```

**Review by bbengfort — APPROVED** [review]

**Pull request merged**